lib/internal/worker.js

```diff
@@ -441,6 +455,20 @@ class Worker extends EventEmitter {
   }
 }
 
+/**
+ * A worker to be used internally by node (ex module/esm/worker). This functions as a siganal to
+ * handle its setup and script in a particular way.
```

Comment:
```suggestion
 * A worker which has an internal module for entry point (e.g. internal/module/esm/worker).
```

Reply:
Let's handle in a follow-up. We might make the cut for v20, but probably not if CI has to run again).